### PR TITLE
[Flutter & iOS SFA] Update docs

### PR DESCRIPTION
**Pull request by AyushBherwani1998**

### CODE REVIEW

File: docs/sdk/core-kit/sfa-flutter/install.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ Add `single_factor_auth_flutter` as a dependency to your `pubspec.yaml`.
 
 ```yaml
 dependencies:
-  single_factor_auth_flutter: ^2.0.1
+  single_factor_auth_flutter: ^4.0.0
```
Comment (shahbaz17) [inline docs/sdk/core-kit/sfa-flutter/install.mdx:16]: The migration says v2 to v3. Is this correct?

File: docs/migration-guides/sfa-flutter-v2-to-v3.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+#### After (v3)
+
+```dart
+final singleFactorAuth = SingleFactAuthFlutter();
+
+singleFactorAuth.init(
+    SFAParams(
+        network: Web3AuthNetwork.mainnet,
```
Comment (shahbaz17) [inline docs/migration-guides/sfa-flutter-v2-to-v3.mdx:37]: Add a line or two related to changes from TorusNetwork to Web3AuthNetwork

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

File: docs/migration-guides/sfa-flutter-v2-to-v3.mdx (hunk 2 above)
Comment (AyushBherwani1998) [reply docs/migration-guides/sfa-flutter-v2-to-v3.mdx:37]: We haven't changed TorusNetwork for Flutter, it's on Android, from v2 itself we were using the Web3AuthNetwork. Let me know if I should add more information.

**Review by shahbaz17 — APPROVED** [review]
lgtm

**Pull request merged**